src/lib/ui/AccessCodeTable/AccessCodeTable.stories.tsx

```diff
@@ -0,0 +1,97 @@
+import { Dialog } from '@mui/material'
+import type { Meta, StoryObj } from '@storybook/react'
+import ms from 'ms'
+import { v4 as uuid } from 'uuid'
+
+import { AccessCodeTable } from 'lib/ui/AccessCodeTable/AccessCodeTable.js'
+
+/**
+ * These stories showcase the device manager.
+ */
+const meta: Meta<typeof AccessCodeTable> = {
+  title: 'Example/AccessCodeTable',
+  component: AccessCodeTable,
+  args: {
+    accessCodes: [
+      {
+        access_code_id: uuid(),
+        type: 'time_bound',
+        code: '1234',
+        created_at: Date.now().toString(),
+        device_id: 'some_device_id',
+        status: 'set',
+        name: 'Guest - Gonzalez',
+        starts_at: new Date(Date.now() - ms('1d')).toISOString(),
+        ends_at: new Date(Date.now() + ms('2d')).toISOString(),
+      },
+      {
+        access_code_id: uuid(),
+        type: 'time_bound',
+        code: '1234',
+        created_at: Date.now().toString(),
+        device_id: 'some_device_id',
+        status: 'set',
+        name: 'Guest - Thompson',
+        starts_at: new Date(Date.now() + ms('1d')).toISOString(),
+        ends_at: new Date(Date.now() + ms('3d')).toISOString(),
+      },
+      {
+        access_code_id: uuid(),
+        type: 'ongoing',
+        code: '1234',
+        created_at: Date.now().toString(),
+        device_id: 'some_device_id',
+        status: 'set',
+        name: 'Guest - Kranz',
+      },
+      {
+        access_code_id: uuid(),
+        type: 'ongoing',
+        code: '1234',
+        created_at: Date.now().toString(),
+        device_id: 'some_device_id',
+        status: 'set',
+        name: 'Sparkle Cleaners',
+      },
+      {
+        access_code_id: uuid(),
+        type: 'ongoing',
+        code: '1234',
+        created_at: Date.now().toString(),
+        device_id: 'some_device_id',
+        status: 'set',
+        name: 'Guest - yang',
+      },
+      {
+        access_code_id: uuid(),
+        type: 'ongoing',
+        code: '1234',
+        created_at: Date.now().toString(),
+        device_id: 'some_device_id',
+        status: 'set',
+        name: 'Astro Plumbing',
+      },
+    ],
+  },
+}
+
+export default meta
+
+type Story = StoryObj<typeof AccessCodeTable>
+
+export const Content: Story = {}
+
+export const InsideModal: Story = {
+  render: (props) => {
+    // Wrap modal/dialog contents in `seam-components` class
+    // to apply styles when rendered in a portal,
+    // which is the the default MUI behavio.
+    return (
+      <Dialog open fullWidth maxWidth='sm'>
+        <div className='seam-components'>
```

Comment:
Will this make integration difficult for our consumers? Do we need to document this or can we find a solution that will wrap the model in the right class?

Reply:
> Will this make integration difficult for our consumers? 

Yea, it does assume the consumer knows the difference between rendering in the DOM tree, and via portal somewhere else. IMO the example code should act as sufficient documentation?

Reply:
> will wrap the model in the right class?

Interesting... we _could_ just add the class to all the modal contents in case they're rendered somewhere else. Having duplicate parent classes of `.seam-components` shouldn't pose issues.

Reply:
https://github.com/seamapi/react/issues/85